monofony/api-bundle/0.1/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "Monofony\\Bundle\\ApiBundle\\MonofonyApiBundle": ["all"]
```

Comment:
I dont understand this bundle. It does not have any code in it...

Reply:
It will contains some controller actions on recipes for Api platform for basic endpoints (register, login, forgotten password...).
But I think I can just create a library instead of a bundle.